packages/vuetify/src/shims.d.ts

```diff
@@ -16,9 +16,12 @@ declare global {
     }
   }
 }
-
+export interface _GlobalComponents {
+  // @generate-components
+}
```

Comment:
```suggestion

interface _GlobalComponents {
  // @generate-components
}

```

Reply:
oh yeah you're right, I forgot to remove the export, thanks

Reply:
@KaelWD Let me know if you need something else before merging. Thank you!